Tests/SuperLinq.Test/HasDuplicatesTest.cs

```diff
@@ -0,0 +1,230 @@
+﻿namespace Test;
+
+public class HasDuplicatesTest
+{
+	[Fact]
+	public void When_Asking_For_Duplicates_On_Sequence_Without_Duplicates_Then_False_Is_Returned()
+	{
+		var stringArray = new[]
```

Comment:
Please follow the pattern in other tests: 
```c#
using var sequence = TestingSequence.Of(
  "FirstElement", 
  "SecondElement",
  "ThirdElement");

var hasDuplicates = sequence.HasDuplicates();

Assert.False(hasDuplicates);
```

NB: `TestingSequence` also exists in `SuperLinq.Async.Test` for async sequences, so you can follow the same pattern for async tests.

> **Note**
> The purpose of `TestingSequence` is to verify that operators use and dispose sequences correctly. `using var sequence = TestingSequence` is a common pattern in SuperLinq as a way to ensure valid behavior on the operator side.